hermes/probe/delete/delete.go

```diff
@@ -88,11 +91,12 @@ func DeleteFile(ctx context.Context, fileID int32, target *probe.Target, client
 		}
 
 		target.LatencyMetrics.APICallLatency[metrics.APIDeleteFile][status].Metric(apiLatency).AddFloat64(time.Now().Sub(start).Seconds())
-		return fileID, fmt.Errorf("delete(%q, %q) failed; status %v: %w", bucket, filename, status, err)
+		return fileID, fmt.Errorf("deleteFile(%q, %q) failed; status %v: %w", bucket, filename, status, err)
```

Comment:
Nit: make it match the name of the function: s/deleteFile/DeleteFile/.
Error messages should start with lower-case, except when the first word is a proper name / identifier.
Here and elsewhere.

Reply:
Ah okay, I didn't realise that identifiers over-ruled the lower-case rule, thanks.

I'll amend this.